db/seeds.rb

```diff
@@ -12,6 +12,25 @@
              password_confirmation: 'password',
              admin: false
 
+Field.create! name: 'Curb cut',
+              category: 'Accessible',
+              field_type: :choice,
+              choices: ["Within 20'", 'No curb cut', 'No curb'],
+              description: 'Determine the border between the road and the bus stop.',
+              rank: 1
+
+Field.create! name: 'Solar lighting',
+              category: 'Technology',
+              field_type: :boolean,
+              description: 'Determine if this stop has solar lighting.',
+              rank: 1
```

Comment:
What's a rank?

Reply:
At what position within a particular category a field should fall.

Reply:
oh! cool